src/Draco.LanguageServer/DracoDocumentRepository.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+
+namespace Draco.LanguageServer;
+
+internal class DracoDocumentRepository
+{
+    public Dictionary<DocumentUri, string> Documents { get; } = new();
```

Comment:
Readonly

Reply:
Do we need to make this public? Why not make a safe accessor method that returns empty string in case the document was not present for some reason?